Document the autounpublish feature (#412)
codewars/docs: content/authoring/tutorials/create-first-kata.mdx

That said, no matter how hard you try, you can never satisfy everyone. Even if your kata receives overwhelmingly positive votes, some users might still not like it. Do not worry about this too much, but do not disregard their feedback either. Give their remarks some consideration, they may point out ways in which your kata can be improved. Listen to everyone, answer their questions, consider suggestions, and fix any issues that crop up. Also, do not rush things too much. If someone says something you are not sure about, just wait for another opinion. Then again, users tend to disagree a lot, and so might offer contradicting suggestions. It can be difficult, but in such cases you need to either find a compromise or just pick one option over the other and move forward. Your mindset and approach to the beta process matter: experience shows that kata whose authors engage actively and constructively in the process tend to graduate out of beta quite quickly, whereas other kata get retired within minutes, or, perhaps worse, never leave beta status due to the author's unwillingness to accept advice or fix issues.

> Through the course of review, your kata could get automatically unpublished after collecting some amount of issues. You are free to fix the issues, mark them as resolved, and publish your kata again. You will not be able to republish your kata until all reported problems are addressed. For more details on auto-unpublish feature see description of the [beta process](/curation/kata#auto-unpublish).

Author: ```suggestion
Through the course of review, your kata could get automatically unpublished after collecting some amount of issues. Automatic unpublishing is not the end of the road for your kata. While back in draft, take note of the issues and fix the underlying problems in your kata. Once you've implemented those, you can resolve the issues and re-publish. You will not be able to republish your kata until all reported problems are addressed. For more details on auto-unpublish feature see description of the [beta process](/curation/kata#auto-unpublish).
```

> Thorough the course of review, your kata could get automatically unpublished after collecting some amount of issues. You are free to fix the issues, mark them as resolved, and publish your kata again. You will not be able to republish your kata until all reported problems are addressed. For more details on auto-unpublish feature see descritpion of the [beta process](/curation/kata#auto-unpublish).

Author: ```suggestion
Thorough the course of review, your kata could get automatically unpublished after collecting some amount of issues. You are free to fix the issues, mark them as resolved, and publish your kata again. You will not be able to republish your kata until all reported problems are addressed. For more details on auto-unpublish feature see description of the [beta process](/curation/kata#auto-unpublish).
```

> Thorough the course of review, your kata could get automatically unpublished after collecting some amount of issues. You are free to fix the issues, mark them as resolved, and publish your kata again. You will not be able to republish your kata until all reported problems are addressed. For more details on auto-unpublish feature see description of the [beta process](/curation/kata#auto-unpublish).

Reviewer: `Thorough` -> `Through`

> Thorough the course of review, your kata could get automatically unpublished after collecting some amount of issues. You are free to fix the issues, mark them as resolved, and publish your kata again. You will not be able to republish your kata until all reported problems are addressed. For more details on auto-unpublish feature see description of the [beta process](/curation/kata#auto-unpublish).

Author: ```suggestion
Through the course of review, your kata could get automatically unpublished after collecting some amount of issues. You are free to fix the issues, mark them as resolved, and publish your kata again. You will not be able to republish your kata until all reported problems are addressed. For more details on auto-unpublish feature see description of the [beta process](/curation/kata#auto-unpublish).
```

> Through the course of review, your kata could get automatically unpublished after collecting some amount of issues. Automatic unpublishing is not the end of the road for your kata. While back in draft, take note of the issues and fix the underlying problems in your kata. Once you've implemented those, you can resolve the issues and re-publish. You will not be able to republish your kata until all reported problems are addressed. For more details on auto-unpublish feature see description of the [beta process](/curation/kata#auto-unpublish).

Author: ```suggestion
Through the course of review, your kata could get automatically unpublished after collecting some amount of issues. Automatic unpublishing is not the end of the road for your kata. While back in draft, take note of the issues and fix the underlying problems in your kata. Once you've implemented those, you can resolve the issues and re-publish. For more details on auto-unpublish feature see description of the [beta process](/curation/kata#auto-unpublish).
```
